RELEASE.md

```diff
@@ -3,6 +3,7 @@
 ## Major features and improvements
 * Added a `MatplotlibS3Writer` dataset in `contrib` for saving Matplotlib images to S3.
 * Added a `JSONGCSDataSet` dataset in `contrib` for connecting to Google Cloud Storage.
+* Added `__sub__` method to Pipeline objects for trimming off subsets of a pipeline.
```

Comment:
Minor, maybe clearer/easier to understand for users:
```suggestion
* Pipelines can be deducted with `pipeline1 - pipeline2` .
```

Reply:
Great suggestion, that is much clearer.